core-services/prow/02_config/_config.yaml

```diff
@@ -496,6 +496,9 @@ plank:
       by [linking to](https://github.com/kubernetes/community/blob/master/contributors/devel/flaky-tests.md#filing-issues-for-flaky-tests)
       an [open issue](https://github.com/{{.Spec.Refs.Org}}/{{.Spec.Refs.Repo}}/issues?q=is:issue+is:open)
       when you hit one in your PR.'
+    integr8ly/integreatly-operator: |-
+      ### Report flaky E2E tests and PROW issues in [INTLY-7197](https://issues.redhat.com/browse/INTLY-7197) epic. Detailed instructions can be found in the description of said epic.
```

Comment:
I don't think that those `###` are necessary at the beginning of the comment. But this is intentional, I fine with it.

Reply:
That is intentional to make the text bigger. :)